src/main/flight/pid.h

```diff
@@ -73,6 +73,7 @@
 
 #ifdef USE_WING
 #define ANGLE_PITCH_OFFSET_MAX 450
+#define S_TERM_CLI_SCALE 0.01f
```

Comment:
What does CLI stand for here ?

Reply:
CLI in configurator/settings.
Like in CLI when user types set s_roll = 50, in the firmare it is divided by 100, so it becomes 0.5

Reply:
Make it `S_TERM_SCALE`. No need to include CLI in the name :)

Reply:
done